### DISCUSSION TITLE
How to subscribe to draging of titlebar

### DISCUSSION THREAD

**Discussion by martinrhan** [AvaloniaUI/Avalonia#12231] [Q&A]
My project has a class derived from Window. 'ExtendClientAreaToDecorationsHint' is set to true in constructor to enable managed title bar. Event handler is added in OnApplyTemplate. I have tried to add handler to PointerPressed event of the TitleBar itself, and PART_MouseTracker, PART_Container, and PART_Background. Non of them have the handler function fired when I try to drag the window.

**Comment by timunie** [comment]
I can't reproduce the issue. does your items have a solid backround (transparent is okay, 'null' is not)

**Reply by martinrhan** [reply]
Which control did you add event handler to PointerPressed ?

**Reply by martinrhan** [reply]
I created a demo and uploaded it to Github, please take a look. https://github.com/martinrhan/TitleBarExperiment

**Reply by timunie** [reply]
I was unable to get it working also. Maybe you want to take a look into how FluentAvalonia did solve it in their AppWindow.
